src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -95,9 +91,12 @@ public void onTestFailure(ITestResult result) {
     public void onTestSkipped(ITestResult result) {
         logger.info(result.getName() + " skipped...");
         try {
-            appendLogToAllure();
-            clearFile();
-        } catch (IOException e) {
+            allureUtil.appendLogToAllure();
+            allureUtil.clearFile();
```

Comment:
is clearing a file part of Allure functionality?

Reply:
fixed